MonticelloTonel-Core.package/TonelParser.class/instance/hasStatefulTraits.st

```diff
@@ -1,5 +1,5 @@
 private testing
 hasStatefulTraits
-	"Pharo has stateful traits starting on version 7"
+	"Pharo has stateful traits starting on version 7, Squeak does not have them at all"
 	
-	^ SystemVersion current major >= 7
\ No newline at end of file
+	^ SystemVersion current baseName ~= 'Squeak' and: [SystemVersion current major >= 7]
```

Comment:
#baseName comes from Etoys-Squeakland and is not available in Pharo. Is there a way that relies only on the basics?

In Pharo there is #type, but not in Squeak. In Squeak there is #isSqueak, but not in Pharo... hmm. Looks like the safest way is to duplicate some of the logic and just check the #version string.